src/qml/PluginManagerSettings.qml

```diff
@@ -14,6 +14,9 @@ Popup {
   y: (parent.height - height) / 2
   padding: 0
 
+  signal unLoadClicked(string Uuid)
+  signal loadClicked(string Uuid)
```

Comment:
Hm, I think it'd be nicer if we implemented something slightly different and have the plugin manager call function _attached to the plugin Item {} itself_ when enabling/disabling a plugin. 

That way we a/ empower plugin authors, and b/ we avoid adding code in qgismobileapp.qml to handle a plugin-specific Uuid handling which I'm not fond of.